tests/contrib/langchain/test_langchain_llmobs.py

```diff
@@ -315,6 +335,58 @@ def test_llmobs_chain_schema_io(self, langchain, mock_llmobs_span_writer, mock_t
         )
         _assert_expected_llmobs_llm_span(trace[1], mock_llmobs_span_writer, mock_io=True)
 
+    @pytest.mark.skipif(sys.version_info < (3, 10, 0), reason="Requires unnecessary cassette file for Python 3.9")
```

Comment:
instead of skipping this for Python 3.9, we can just use both sets of cassette files we have, i.e.
```python
cassette_name="openai_embedding_query_39.yaml" if PY39 else "openai_embedding_query.yaml",
```